common/src/main/java/com/redhat/cloud/notifications/models/EndpointType.java

```diff
@@ -10,7 +10,9 @@ public enum EndpointType {
     @JsonProperty("email_subscription")
     EMAIL_SUBSCRIPTION(false),
     @JsonProperty("camel")
-    CAMEL(true);
+    CAMEL(true),
+    @JsonProperty("eda")
```

Comment:
@jeromemarc internally the type is gonna be "eda". Do we want to use a longer name?

Reply:
I've decided to go with "ansible". It's better than an abbreviated EDA, and the abbreviation can likely change.